src/components/PathwayCourse/index.js

```diff
@@ -106,36 +146,156 @@ function PathwayCourse() {
   const pathwayCourseData = pathways.find((item) => {
     return item.id == pathwayId;
   });
+  useEffect(() => {
+    console.log(
+      upcomingBatchesData,
+      userEnrolledClasses,
+      pathwayCourse?.data?.courses,
+      "Here"
+    );
+  }, [upcomingBatchesData, userEnrolledClasses]);
+
+  const handleVideo = () => {
+    return <YouTube className={"youtube-video"} videoId="DDFvJmC3J5M" />;
+  };
```

Comment:
This could be removed since it won't be used now.

Reply:
@amansharmma